src/utils/backupTarget.js

```diff
@@ -0,0 +1,7 @@
+// get the available backup targets
+export function getAvailableBackupTargets(backupTarget) {
+  if (!backupTarget || !backupTarget.data || backupTarget.data.length === 0) {
+    return []
+  }
+  return backupTarget.data.filter((item) => item.available)
+}
```

Comment:
Will it be clearer to show all targets with unavailable ones grayed out?
Hiding unavailable targets might confuse users about why some are missing.

Reply:
Updated. 
<img width="802" alt="image" src="https://github.com/user-attachments/assets/e68475f1-b441-4456-9ce0-6e871cc42cca" />